### DISCUSSION TITLE
New Install 401 With X-Api-Key

### DISCUSSION THREAD

**Discussion by BlythMeister** [DependencyTrack/dependency-track#1971] [Q&A] [answered]
I have tried to follow the setup guide.

I am running with `docker-compose`

i get the api key from the automation team and then run 

`curl -v 'GET' 'https://localhost:8081/api/v1/user/self' -H 'accept: application/json' -H 'X-Api-Key: [key-here]'`

the response is 

```
* Could not resolve host: GET
* Closing connection 0
curl: (6) Could not resolve host: GET
*   Trying 127.0.0.1:8081...
* Connected to localhost (127.0.0.1) port 8081 (#1)
> GET /api/v1/user/self HTTP/1.1
> Host: localhost:8081
> User-Agent: curl/7.83.1
> accept: application/json
> X-Api-Key: zLEebvYvw7vhdGUB0PGQ3JiyC8O3pbRL
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 401 Unauthorized
< X-Powered-By: Dependency-Track v4.5.0
< Access-Control-Allow-Origin: *
< Access-Control-Allow-Methods: GET, POST, PUT, DELETE, OPTIONS
< Access-Control-Allow-Headers: Origin, Content-Type, Authorization, X-Requested-With, Content-Length, Accept, Origin, X-Api-Key, X-Total-Count, *
< Access-Control-Expose-Headers: Origin, Content-Type, Authorization, X-Requested-With, Content-Length, Accept, Origin, X-Api-Key, X-Total-Count
< Access-Control-Allow-Credentials: true
< Access-Control-Max-Age: 3600
< Content-Length: 0
<
* Connection #1 to host localhost left intact
```

I am trying to work out if i have done something wrong or am missing something.

I can see through fiddler that the UI is able to access those api (using a bearer token)
From the logging, i cannot see anything in the logs which hint to the issue

**Comment by syalioune** [comment, marked as answer]
Hello @BlythMeister 
That API /api/v1/user/self is to get the current logged in user for GUI. Since you are using an API Key, it is "normal" to get a 401 since you are not logged in as an end user. Have you tried other API and what is your endgame 😃 ?

**Reply by BlythMeister** [reply]
I was having other issues.  Tried that API call..but your right, other APIs work.

That's 2 hours I'm never getting back!
